yjit/src/codegen.rs

```diff
@@ -5911,6 +6000,10 @@ fn jit_rb_str_concat(
     // If the peeked-at compile time argument is something other than
     // a string, assume it won't be a string later either.
     let comptime_arg = jit.peek_at_stack(&asm.ctx, 0);
+    if unsafe { RB_TYPE_P(comptime_arg, RUBY_T_FIXNUM) } {
+        return jit_rb_str_concat_codepoint(jit, asm, _ocb, _ci, _cme, _block, _argc, _known_recv_class);
+    }
```

Comment:
Minor nitpick that the underscore by convention e.g. `_ocb` means that an argument won't be used, so you shouldn't be passing or using arguments named `_something`. If you do use them, you should remove the underscore from the name.

Reply:
Thanks. I wasn't sure if passing arguments forward constituted being used. It's an easy enough fix. Thanks.